core/server/models/app.js

```diff
@@ -34,7 +34,7 @@ App = ghostBookshelf.Model.extend({
     * @return {Array} Keys allowed in the `options` hash of the model's method.
     */
     permittedOptions: function permittedOptions(methodName) {
-        var options = ghostBookshelf.Model.permittedOptions(),
+        var options = ghostBookshelf.Model.permittedOptions.call(this, methodName),
```

Comment:
General: There are some more models to update, or? 🤔 
Why did you only update these three?

Reply:
missed the others - will update :relaxed:

Reply:
updated btw!

Reply:
👍